### ISSUE TITLE
Browser selection

### ISSUE THREAD

**Issue by kasajian** [stgolem/BrowserSelector#1]
Are you open to the idea of a feature the probes the system to discover already installed browsers?

Although there's no full-proof way to do it, there is some precedence in doing this in the automated testing market that web developers use to test their web-sites.  For instance, Karma code shows some examples of how they locate the various browsers, and it seems like those developers are eager to support new browsers.

For instance, here's an example of how to do it for Chrome:
https://stackoverflow.com/questions/45384893/how-do-i-use-c-sharp-to-get-the-path-to-chrome-exe-on-windows/45384927#45384927

**Comment by stgolem** [comment]
Yes! This is on my roadmap. I've posted some thoughts in the original repo https://github.com/DanTup/BrowserSelector/issues/10 

Recently I had not much time to develop this little project. 
I am now using it as main tool at my PC. For me this helps a lot!
And I can add some crazy browsers with simple config, like TorBrowser for safety browse. 
Or open something with Firefox different profile. 
Or even in new window directly. 

Feel free to open a PR with your ideas here.

**Comment by Andy-JT** [comment]
Hi

I've gone through the set-up steps, and set BrowserSelector as the default browser. However, testing the 'default config' URLs via Start >>> Run I do not get the expected results, i.e. https://visualstudio.com is opening in Chrome (which I did not expect), I changed the config to have *.visualstudio.com but that did not work. However, using www.visualstudio.com does launch in Edge. Am I seeing expected behaviour ? - I'm using Windows 10 Enterprise Version 1709. 

This would be a great tool for our environment, as we need to have Edge (and IE via Enterprise Mode) as our default but get constant demand for some browser based apps (and vendors) insist on Chrome

**Comment by stgolem** [comment]
@Andy-JT Sorry for late answer. 
As you can read in Readme: https://github.com/stgolem/BrowserSelector#urls i have disabled all URL matching logic in this fork for now. 

I'm planning to add this later with more features. 
Sadly, i do not have much time for this project now.

**Comment by Andy-JT** [comment]
okay thanks very much for the reply. I'll look forward to the later update - when you get time to look at it :-)
